### PR TITLE
Use Bzlmod lockfile locally to support offline builds

### PR DESCRIPTION
**Pull request by brentleyjones**

### CODE REVIEW

File: bazel_6.bazelrc [hunk 1]
```diff
@@ -7,6 +7,3 @@ build --enable_bzlmod
 build --apple_crosstool_top=@local_config_apple_cc//:toolchain
 build --crosstool_top=@local_config_apple_cc//:toolchain
 build --host_crosstool_top=@local_config_apple_cc//:toolchain
-
-# Disabled until bugs are fixed
```
Comment (mattrobmattrob) [inline bazel_6.bazelrc:11]: The bugs were fixed, I guess?

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,7 @@
 .DS_Store
 
 /bazel-*
+MODULE.bazel.lock
```
Comment (mattrobmattrob) [inline .gitignore:4]: This will include the `examples/*/MODULE.bazel.lock` as well, correct?

**Review by mattrobmattrob — APPROVED** [review]

File: bazel_6.bazelrc (hunk 1 above)
Comment (brentleyjones) [reply bazel_6.bazelrc:11]: It's disabled by default in 6.4.0, so no need to call it out here. Didn't want to see if they actually were fixed and enable it.

File: .gitignore (hunk 2 above)
Comment (brentleyjones) [reply .gitignore:4]: Yep!

**Pull request merged**